index.html

```diff
@@ -329,7 +329,8 @@ <h2><a>LDP-RS</a>s</h2>
           <p>
             Responses to <code>GET</code> requests that apply a <code>Prefer</code> request header to any <a>LDP-RS</a>
             MUST include the <code>Preference-Applied</code> response header as defined in [[!RFC7240]]
-            <a href='https://tools.ietf.org/html/rfc7240#section-3'>section 3</a>.
+            <a href='https://tools.ietf.org/html/rfc7240#section-3'>section 3</a> if and only all specified parameters
```

Comment:
missing the second if in 'if and only if'

Reply:
ha, sry didn't mean to say that twice.  fooled by browser caching... plus i'm an idiot.

Reply:
thanks... you ain't the idiot that missed out the 'if' ;-)